### PR TITLE
fix: do not ignore empty plural values

### PR DESCRIPTION
**Pull request by vonovak**
# Description

It's a corner case but I think the test case explains it

## Types of changes

[//]: # (What types of changes does your code introduce to Lingui?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

Fixes # (issue)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/lingui/js-lingui/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT](https://github.com/lingui/js-lingui/blob/main/CODE_OF_CONDUCT.md) docs
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/core/src/compile/compileMessage.test.ts [hunk 1]
```diff
@@ -83,9 +92,9 @@ describe("compile", () => {
   it("should support nested choice components", () => {
     const cache = prepare(
       `{
-      gender, select, 
-      male {{numOfGuests, plural, one {He invites one guest} other {He invites # guests}}} 
-      female {{numOfGuests, plural, one {She invites one guest} other {She invites # guests}}} 
+      gender, select,
```
Comment (vonovak) [inline packages/core/src/compile/compileMessage.test.ts:95]: this happened after reformatting with prettier
